core/src/main/java/com/linecorp/armeria/server/annotation/ExceptionHandlingFunction.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2017 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.annotation;
+
+import com.linecorp.armeria.common.HttpRequest;
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.HttpStatus;
+import com.linecorp.armeria.common.RequestContext;
+
+/**
+ * An interface for exception handler.
+ */
+@FunctionalInterface
+public interface ExceptionHandlingFunction {
```

Comment:
Which is better? `ExceptionHandlingFunction` vs. `ExceptionHandlerFunction` /cc @anuraaga :-)

Reply:
I will rename it as `ExceptionHandlerFunction` because I removed `ExceptionHandling` annotation while changing exception map to list.